### ISSUE TITLE
enhancing doc around resource pool settings w.r.t standalone managed esxi hosts

### ISSUE THREAD

**Issue by bby-bishopclark** [chef/knife-vsphere#494]
I struggled a bit yesterday and today, trying to land a VM on a standalong esxi host which *was* managed through vCenter.  The value for --resource-pool was really tricky to nail.

The problem?  My solo managed hosts are in a folder.

I'd love to write/see a mod to the docco around that command so when we run into that issue again, we won't have completely forgotten and get jammed up again.

The Resource-pool specification for a managed solo (non-cluster) host with no actual Resource Pools is 

```
[FolderName/]HostFQDN
```
eg
```
--resource-pool Solos/han.rebel.net 
```

That SEEMS to have worked for me.  I can't confirm whether or what works for an UNmanaged solo ESXi host (or whether the plugin will even interact!)

The current readme says 
```
--resource-pool POOL|CLUSTER - The resource pool into which to put the cloned VM. Also accepts a cluster name.
```

Anyway, that's the thinking, and I'm all explained out for such a simple thing.  Can or should the doc reflect that syntactical challenge?

**Comment by swalberg** [comment]
Hi, yes, an improvement to the docs would be most welcomed. Doesn't need to be big.

We ran into some other things in #244 with unmanaged hosts that make it less than helpful, so this plugin hasn't tried to address those.

**Comment by bby-bishopclark** [comment]
Nah, it's not something that's gotta be addressed specifically by the plugin;  it's just an idiosyncracy of the API itself, and how it needs that string to be sent.  We could try and coddle it a bit, but that's above-and-beyond.

I'll try and figure out something long enough to describe this corner case and small enough as befitting its rarity.
